app/src/main/kotlin/com/flxrs/dankchat/data/twitch/message/ClearChatMessage.kt

```diff
@@ -33,4 +37,19 @@ data class ClearChatMessage(
             )
         }
     }
-}
\ No newline at end of file
+
+
+}
+
+// Extracted from ChatAdapter, TODO localize
+val ClearChatMessage.fullText: String
```

Comment:
I would like these extensions to have more descriptive names. Here we could use something like `systemMessage`